### ISSUE TITLE
** (Mix) The task "sentinel.gen.views" could not be found

### ISSUE THREAD

**Issue by sgeos** [britton-jb/sentinel#47]
Maybe I missed something important, but the instructions and source code seem to indicate the mix task `sentinel.gen.views` should exist.  I get the following error when I try to use it.
```sh
** (Mix) The task "sentinel.gen.views" could not be found
```

The task does not exist in `deps/sentinel/lib/mix/tasks/`.

**Comment by britton-jb** [comment]
Currently master and the latest hex release have diverged, I'm working on a few more changes before the next release because the next hex release will contain breaking changes, and I want to get all of the breaking changes that are on the immediate horizon out of the way. Sorry about the confusion.

**Comment by sgeos** [comment]
Thank you for the clarification.  It probably makes sense to use a feature branch for divergent development in the future.  Putting a warning at the top of the README is another alternative, depending on how you want to run the project.

**Comment by britton-jb** [comment]
Yeah, in the future master will remain up to date with the latest hex release, and a dev or version branch will contain the changes. Again, sorry about the confusion.
